### PR TITLE
Add NUR support

### PR DESCRIPTION
**Pull request by diogox**

### CODE REVIEW

**Comment by nrdxp** [comment]
Keep in mind that you can force push to the branch your PR is based off of if you need to rebase or make changes. Making duplicate pull requests isn't ideal. That said, it looks good to me. I'm wondering if we need to specifically declare NUR as an input though, since it is already part of Nix registry.

I'll give it a try and see how it works before merging.

**Comment by diogox** [comment]
Sorry, I wasn't sure how I could sync a fork with the original, so it was just easier to make a new fork and PR.

Do give it a try. It works for me, but I don't quite understand everything in Nix yet, let alone flakes :smile:

File: flake.nix [hunk 1]
```diff
@@ -6,9 +6,10 @@
       master.url = "nixpkgs/master";
       nixos.url = "nixpkgs/release-20.09";
       home.url = "github:nix-community/home-manager/release-20.09";
+      nur.url = "github:nix-community/NUR";
```
Comment (nrdxp) [inline flake.nix:9]: Remove this line. NUR lives in the flake registry, so we'll just pull it from there.

**Review by nrdxp — CHANGES_REQUESTED** [review]
Overall good job. Just remove the explicit import and it's ready to merge.

File: flake.nix (hunk 1 above)
Comment (diogox) [reply flake.nix]: Done :+1:

**Comment by diogox** [comment]
Btw, I'm trying to add an `unstable` overlay to my config, but haven't been able to figure out how to do that. Do you know how to do that?

**Review by nrdxp — APPROVED** [review]
Thank you.

**Pull request merged**